src/node_crypto.cc

```diff
@@ -4779,14 +4783,17 @@ void DiffieHellman::Initialize(Environment* env, Local<Object> target) {
   env->SetProtoMethod(t, "setPublicKey", SetPublicKey);
   env->SetProtoMethod(t, "setPrivateKey", SetPrivateKey);
 
-  t->InstanceTemplate()->SetAccessor(
+  Local<FunctionTemplate> verify_error_getter_templ =
+      FunctionTemplate::New(env->isolate(),
+                            DiffieHellman::VerifyErrorGetter,
+                            Local<Value>(),
```

Comment:
Would  `env->as_external()` be appropriate here as well? Or better yet, is there a place where it isn't appropriate (there are 10 places still using it)?

Reply:
I guess anywhere where as_external was previously used. I'll get on it.

Reply:
👍 Done.